sdk/elixir/runtime/main.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	"github.com/iancoleman/strcase"
+)
+
+const (
+	ModSourceDirPath     = "/src"
+	sdkSrc               = "/sdk"
+	genDir               = "sdk"
+	schemaPath           = "/schema.json"
+	defaultElixirVersion = "1.16.1-erlang-26.2.2-debian-bookworm-20240130-slim"
```

Comment:
Best if full image ref, including the digest as well.

Reply:
Fixed in https://github.com/dagger/dagger/pull/6967/commits/a644d96ed3a67dbc2dac46ad64bfd39bcaa32acf